index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Tribute Page</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+
+    <div id="img-div">
+      <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt=""
+        id="image">
+      <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase
```

Comment:
I'd suggest the usage of a <p> tag over a <div> here, according to HTML best practices, since this is a description.